jstests/aggregation/isodates.js

```diff
@@ -0,0 +1,243 @@
+/*
```

Comment:
I will look at [jstests/aggregation/bugs/server19548.js](https://github.com/mongodb/mongo/blob/r3.3.2/jstests/aggregation/bugs/server19548.js) and see what I can do, but please be patient, It was kind if tedious to write these tests, and yes they were copied from testall.js 😉 I always try to copy the style of the existing code when contributing to open source. Sadly I took the wrong example now. But you're absolutely right about readability of tests, so I will look into it and see what I can do.